core/controllers/collection_viewer.py

```diff
@@ -87,11 +88,41 @@ def get(self, collection_id):
         try:
             collection_dict = (
                 collection_services.get_learner_collection_dict_by_id(
-                    collection_id, self.user_id,
-                    allow_invalid_explorations=allow_invalid_explorations))
+                    collection_id, self.user_id))
         except Exception as e:
             raise self.PageNotFoundException(e)
 
+        collection_is_public = rights_manager.is_collection_public(
+            collection_id)
+
+        # Insert an 'exploration' dict into each collection node, where the
+        # dict includes meta information about the exploration (ID and title).
+        for collection_node in collection_dict['nodes']:
+            exploration_id = collection_node['exploration_id']
+            exploration_is_private = rights_manager.is_exploration_private(
+                exploration_id)
+            # Temporary variable to avoid line-too-long lint error
+            x = summary_services.get_displayable_exp_summary_dicts_matching_ids(
```

Comment:
Ah, looks like this is just unavoidable. In that case I suggest using  `# pylint: disable=line-too-long` instead (see elsewhere in codebase).